lime/tools/helpers/CSHelper.hx

```diff
@@ -0,0 +1,301 @@
+			
+		}
+		
+		file.writeString ('end libs\n');
+		file.close();
+		
+	}
+	
+	public static function addNativeLibraries (txtPath:String, libPath:String, libraries:Array<NDLL>, architectures:Array<Architecture>) {
+		
+		if (libraries.length == 0) {
+			
+			return;
+			
+		}
+		
+		var file = File.append (txtPath, false);
+		file.writeString ('\nbegin native_libs\n');
+		
+		for (arch in architectures) {
+			
+			var archName = getAndroidABIName (arch);
+			
+			for (lib in libraries) {
+				
+				file.writeString (FileSystem.absolutePath(libPath + "/" + archName + "/" + "lib" + lib.name + ".so").replace("/", "\\") + '\n');
+				
+			}
+			
+		}
+	
+		file.writeString ('end native_libs\n');
+		file.close();
+		
+	}
+	
+	public static function addAndroidABIs (txtPath:String, architectures:Array<Architecture>) {
+		
+		if (architectures.length == 0) {
+			
+			return;
+			
+		}
+		
+		var file = File.append (txtPath, false);
+		file.writeString ('\nbegin android_abis\n');
+		
+		for (arch in architectures) {
+			
+			var archName = getAndroidABIName (arch);
+			file.writeString (archName + '\n');
+			
+		}
+	
+		file.writeString ('end android_abis\n');
+		file.close();
+		
+	}
+		
+	
+	public static function addAssets (txtPath:String, assets:Array<String>) {
+		
+		if (assets.length == 0) {
+			
+			return;
+			
+		}
+		
+		var file = File.append (txtPath, false);
+		file.writeString ('\nbegin android_assets\n');
+		
+		for (asset in assets) {
+			
+			file.writeString (FileSystem.absolutePath(asset).replace("/", "\\") + '\n');
+			
+		}
+		
+		file.writeString ('end android_assets\n');
+		file.close();
+		
+	}
+	
+	public static function addGUID (txtPath:String, guid:String) {
+		
+		var file = File.append (txtPath, false);
+		file.writeString ('\nbegin guid\n');
+		
+		file.writeString ('$guid\n');
+		
+		file.writeString ('end guid\n');
+		file.close ();
+		
+	}
+	
+	public static function compile (project:HXProject, path:String, outPath:String, arch:String, platform:String, buildFile:String = "hxcs_build.txt", noCompile:Bool = false) {
+		
+		var args = [ "run", project.config.getString ("cs.buildLibrary", "hxcs"), buildFile, "--arch", arch, "--platform", platform, "--out", outPath, "--unsafe" ];
+		if (noCompile)
+			args.push ("--no-compile");
+		var code = ProcessHelper.runCommand (path, "haxelib", args);
+		
+		if (code != 0) {
+			
+			Sys.exit (code);
+			
+		}
+		
+	}
+	
+	public static function buildGradleProj (path:String) {
+		
+		var gradlePath = FileSystem.absolutePath(path + "/" + "gradlew");
+		ProcessHelper.runCommand (path, gradlePath, ["build", "assembleRelease"]);
+		
+	}
+	
+	inline public static function buildSln (path:String, slnPath:String, task:String = null) {
+		
+		buildCSProj (path, slnPath, task);
+		
+	}
+	
+	public static function buildCSProj (path:String, csprojPath:String, task:String = null) {
+		
+		var msBuildPath = "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe";
```

Comment:
Probably I should use `ProcessHelper.runCommand ("", "msbuild", params)` instead?